ufork-wasm/www/index.js

```diff
@@ -1374,201 +468,102 @@ $snapshotButton.onclick = function download_snapshot_file() {
+            const sponsor = core.u_read_quad(event.t);
+            sponsor.y = core.u_fixnum(num);
+            core.u_write_quad(event.t, sponsor);
+            draw_host();
         }
     }
 };
 
-let wasm_call_in_progress = false;
-function wasm_mutex_call(wasm_fn) {
-    return (...args) => {
-        if (wasm_call_in_progress) {
-            console.log("ERROR! re-entrant WASM call", wasm_fn, args);
-            throw new Error("re-entrant WASM call");
-        }
-        try {
-            wasm_call_in_progress = true;  // obtain "mutex"
-            return wasm_fn(...args);
-        } finally {
-            wasm_call_in_progress = false;  // release "mutex"
-        }
-    }
-}
-
-function test_suite(exports) {
-    console.log("u_fixnum(0) =", u_fixnum(0), u_fixnum(0).toString(16), u_print(u_fixnum(0)));
-    console.log("u_fixnum(1) =", u_fixnum(1), u_fixnum(1).toString(16), u_print(u_fixnum(1)));
-    console.log("u_fixnum(-1) =", u_fixnum(-1), u_fixnum(-1).toString(16), u_print(u_fixnum(-1)));
-    console.log("u_fixnum(-2) =", u_fixnum(-2), u_fixnum(-2).toString(16), u_print(u_fixnum(-2)));
-    console.log("h_rom_top() =", h_rom_top(), u_print(h_rom_top()));
-    console.log("h_ram_top() =", h_ram_top(), u_print(h_ram_top()));
-    console.log("u_ramptr(5) =", u_ramptr(5), u_print(u_ramptr(5)));
-    console.log("u_ptr_to_cap(u_ramptr(3)) =", u_ptr_to_cap(u_ramptr(3)), u_print(u_ptr_to_cap(u_ramptr(3))));
-    console.log("u_memory() =", u_memory());
-
-    const rom_ofs = u_rom_ofs();
-    const rom = new Uint32Array(u_memory(), rom_ofs, (u_rawofs(h_rom_top()) << 2));
-    console.log("ROM:", rom);
-
-    const ram_ofs = u_ram_ofs();
-    const ram = new Uint32Array(u_memory(), ram_ofs, (u_rawofs(h_ram_top()) << 2));
-    console.log("RAM:", ram);
-
-    const blob_ofs = u_blob_ofs();
-    const blob = new Uint8Array(u_memory(), blob_ofs, u_fix_to_i32(h_blob_top()));
-    console.log("BLOB:", blob);
-
-    const decoded = {
-        space: {origin: [-40, -200], extent: [600, 460]},
-        shapes: [
-            {origin: [5, 3], extent: [21, 13]},
-            {origin: [8, 5], extent: [13, 8]}
-        ]
-    };
-    const encoded = OED.encode(decoded);
-    const enc_lite = oed.encode(decoded);
-    console.log("OED encoded:", encoded, enc_lite);
-    let dec_encoded = OED.decode(encoded);
-    let dec_enc_lite = OED.decode(enc_lite);
-    let dec_lite_encoded = oed.decode(encoded);
-    let dec_lite_enc_lite = oed.decode(enc_lite);
-    console.log("OED decoded:", dec_encoded, dec_enc_lite, dec_lite_encoded, dec_lite_enc_lite);
-    let dec_at11_encoded = OED.decode(encoded, undefined, 11);
-    let dec_at11_enc_lite = oed.decode({ octets: enc_lite, offset: 11 });
-    console.log("OED seek:", dec_at11_encoded, dec_at11_enc_lite);
-}
-
-const wasm_source = "../target/wasm32-unknown-unknown/release/ufork_wasm.wasm";
-//const wasm_source = "../target/wasm32-unknown-unknown/debug/ufork_wasm.wasm";
```

Comment:
The alternatives here remind us how to select the debug build generated by `dev-build.sh`. How else would you recommend encoding this option?

Reply:
I have been getting by replacing "release" with "debug" in the path. Shall I revert to a comment-based solution?

Reply:
Wait. Considering it's a debugger, shouldn't it always use the debug build?

Reply:
I think the comment switching method is a good reminder of what has to change.

We don't want it to _always_ be debug because we don't always **build** for debug, so it might be out-of-date.

Reply:
Makes sense. We build for release in development so we can see if the size of the WASM jumps unexpectedly, right?

Reply:
Correct. The current image is ~40Kb and I want to know when/if it jumps to 44Kb (or higher).